nucleus/payara-modules/hazelcast-bootstrap/src/main/java/fish/payara/nucleus/hazelcast/HazelcastCore.java

```diff
@@ -454,6 +451,7 @@ private void buildNetworkConfiguration(Config config) throws NumberFormatExcepti
             port = Integer.valueOf(configuration.getDasPort());
         }
         config.getNetworkConfig().setPort(port);
+        config.getNetworkConfig().setPortAutoIncrement("true".equalsIgnoreCase(configuration.getPortAutoIncrement()));
```

Comment:
Note: I am not a 100% sure we always should forward the setting from the config but on the other hand I can't see a reason to not do it. It would be interesting to know why we previously only set port auto increment to true for `multicast`.